osu.Framework/IO/Stores/CachedOnlineStore.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using osu.Framework.Extensions;
+using osu.Framework.Platform;
+
+namespace osu.Framework.IO.Stores
+{
+    /// <summary>
+    /// An <see cref="OnlineStore"/> with file system caching
```

Comment:
```suggestion
    /// An <see cref="OnlineStore"/> with file system caching.
```

missing full stop in many other cases too.